src/crypto/public_key/elliptic/curves.js

```diff
@@ -182,14 +182,14 @@ Curve.prototype.keyFromPublic = function (pub) {
 
 Curve.prototype.genKeyPair = async function () {
   let keyPair;
-  if (webCrypto && this.web) {
```

Comment:
Was this a big fix? If so and webCrypto and nodeCrypto variables aren't used anymore you should delete them from the top of the file too.